colab/colab_utils.py

```diff
@@ -0,0 +1,1048 @@
+#@title Please run this cell first!
```

Comment:
Is this file the source of the notebook, or a regular python module? If the latter, please delete these "#@" notebook annotations.

Reply:
This is indeed the regular python module for the specific use of the Colab notebook. The deletions will be done, thank you